packages/itwinui-css/src/checkbox/checkbox.scss

```diff
@@ -16,8 +16,8 @@
     gap: var(--iui-size-xs);
 
     &.iui-loading {
-      cursor: wait;
-      color: var(--iui-color-foreground-4);
+      cursor: progress;
+      font-style: italic;
```

Comment:
Was focusing on changing colors so we pass contrast scores.  Decided to remove the muted text color here & apply italics to align with [tile's loading state](https://github.com/iTwin/iTwinUI/blob/f384be3406512a11b486c583aca14bb427779138/packages/itwinui-css/src/tile/tile.scss#L109).  Also aligned the `cursor`.  Maybe should align the class name as well.

Reply:
Reverted this change, introduced this change in #765.